docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst

```diff
@@ -1,6 +1,96 @@
 .. _dataset-creation-azure-storage:
 
-Create a dataset from azure blob storage
-----------------------------------------
 
-TODO: Add the tutorial how to create a Lightly dataset from a azure bucket here.
+Create a dataset from Azure blob storage
+-----------------------------------------
+
+Lightly allows you to configure a remote datasource like Microsoft's Azure blob storage.
+In this guide, we will show you how to setup your Azure blob storage container and configure your dataset to use said container.
+
+One decision you need to make first is whether you want to use thumbnails.
+Using thumbnails makes the Lightly Platform more responsive, as it's enough to
+load the thumbnails instead of the full images in many cases.
+As a drawback, the thumbnails will be stored in your bucket and thus need storage.
+You have the following three options:
+
+
+- You want to use thumbnails, but don't have them yet. Then you need to give
+  Lightly write access to your bucket to create the thumbnails there for you.
```

Comment:
How do I distinguish between read/write access to the bucket when setting it up? If I cannot, leave it away here.